include/multi_blas_helper.cuh

```diff
@@ -30,7 +101,7 @@ namespace quda
        when we have a multi-1d kernel and the coefficients are stored
        in the functor.
     */
-    constexpr int max_N_multi_1d() { return 32; }
+    constexpr int max_N_multi_1d() { return 24; }
```

Comment:
Same comment as above, should this be abstracted per-target?

Reply:
This limit is more of an algorithmic setting and not target specific.  If there's any need, we could revert this back to 32.